renku_notebooks/util/kubernetes_.py

```diff
@@ -100,3 +100,13 @@ def make_server_name(namespace, project, branch, commit_sha):
     return "{project}-{hash}".format(
         project=project[:54], hash=md5(server_string.encode()).hexdigest()[:8]
     )
+
+
+def make_pvc_name(safe_username, namespace, project, branch, commit_sha):
```

Comment:
do we need two functions here for names? I think the reason `make_server_name` doesn't explicitly include the username is because the username is a part of the jupyterhub pod template - could we not change that so the template doesn't include the username, but then have just one function here that generates the unique string? We can then just attach `-pvc` to it to make it easier to identify.

Reply:
Yeah I know this sucks... But the server name is already being used in the annotation for each user session. So if I change the server name then I have to add a fair amount of code and if statements so that the server class is backwards compatible with the old server names that do not include the user name. 

I made a mistake in making the server name like that - it should have included the user name. So this was the easy way out so that I have to make the server class backwards compatible.